source/DoubleCache/ExistingItemSubscribingCache.cs

```diff
@@ -0,0 +1,112 @@
+﻿using System;
+using System.Collections.Concurrent;
+using System.Threading.Tasks;
+
+namespace DoubleCache
+{
+    public class ExistingItemsSubscribingCache : ICacheAside
```

Comment:
I wonder if eagerly filling the in-memory cache is that useful. What I have done for caching in the past is whenever an item changes (eg: I received a Service Bus message from another service) I just remove it from the cache. If-and-when something requests that item then we can fetch it again, but until it's actually needed that item is just taking up space.